### ISSUE TITLE
Add support for multiple json naming definition

### ISSUE THREAD

**Issue by gabfssilva** [iheartradio/play-swagger#284]
It seems like play-swagger only supports camelCase (since it's the default scala naming convention). It'd be good to define another naming definition since not always people use camelCase, like:

build.sbt

```scala
//namingDefinition := iheartradio.naming.SnakeCase
//namingDefinition := iheartradio.naming.KebabCase
//namingDefinition := iheartradio.naming.UpperCase
//namingDefinition := iheartradio.naming.None 
//no need for CamelCase since it's default in scala naming
namingDefinition := iheartradio.naming.SnakeCase
```

**Comment by gabfssilva** [comment]
Fixed for Play 2.6.x and 2.7x in #285 and #286.

**Comment by kailuowang** [comment]
Thanks @gabfssilva , it's released in 0.9.0 and 0.9.0-PLAY2.7
